lib/importers/tp/saver.rb

```diff
@@ -67,8 +72,17 @@ class IncludedCallOutcome
           'REFER TO CIT-A ENGLAND AND WALES'
         ].freeze
 
-        def satisfied?(call)
-          INVALID_OUTCOMES.exclude?(call.outcome.upcase)
+        INVALID_CALL_OUTCOMES = [
+          'TEST CODE'
+        ].freeze
+
+        def satisfied?(filter_type, call)
+          case filter_type
+          when :where_did_you_hear
```

Comment:
Sorry Ben, I looked at this after your first comment and just saw what I expected to see. 

I have now fixed the code to use the correct constant. Will give some thought to this overnight as feel this could/should have a test.